modin/engines/ray/pandas_on_ray/axis_partition.py

```diff
@@ -70,5 +70,5 @@ class PandasOnRayRowPartition(PandasOnRayAxisPartition):
 
 
 @ray.remote
-def deploy_ray_func(func, *args):
+def deploy_ray_func(func, *args):  # pragma: no cover
```

Comment:
Why are we skipping this function in the code coverage? Is it because codecov does not realize that we reached the code if it is a ray task? If so, can there be inline documentation to mention this?